pylintrc

```diff
@@ -335,19 +326,19 @@ max-args=20
 
 # Argument names that match this expression will be ignored. Default to name
 # with leading underscore
-ignored-argument-names=_.*
+ignored-argument-names=^_
 
 # Maximum number of locals for function / method body
-max-locals=20
+max-locals=100
```

Comment:
Reduced to 60. If code looks generally untidy, the reviewer will catch this at manual inspection anyway.

The rationale here in general is to make our linter useful instead of ignored.